.github/workflows/build.yml

```diff
@@ -0,0 +1,13 @@
+name: Build
+on: [pull_request, workflow_dispatch]
+jobs:
+  cocoapods:
+    name: CocoaPods
+    runs-on: macOS-11
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Use Xcode 13
+        run: sudo xcode-select -switch /Applications/Xcode_13.0.app
```

Comment:
Maybe a dumb question : What machine is actually running this job? If it's a local machine, would we all need to make sure our Xcode installs are named the same / stored in the same directory?

Reply:
Good question.

So, GitHub Action's macOS-11 image provides Xcode images as well (see docs [here](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md#xcode)). So this isn't a local Xcode, but the path that the GH Actions docs want us to specify to use their hosted Xcode.